.gitignore

```diff
@@ -15,3 +15,4 @@ dist
 
 # Runtime data
 coverage
+lib/__snapshots__
```

Comment:
Please remove and read https://facebook.github.io/jest/docs/snapshot-testing.html 👍 

**TL;DR** The idea is to keep those snapshots.